saleor/order/models.py

```diff
@@ -153,6 +153,8 @@ class Order(ModelWithMetadata, ModelWithExternalReference):
         null=True,
         on_delete=models.SET_NULL,
     )
+    draft_save_billing_address = models.BooleanField(null=True, blank=True)
+    draft_save_shipping_address = models.BooleanField(null=True, blank=True)
```

Comment:
If this part is confusing, maybe it's worth to add comment explaining that this flag only should exist in draft order and in Order is null?

Reply:
Comment added